nasa-gibs/worldview
# Update IMERG to version 7

## Description

Fixes #WV-3231 .

Update IMERG Precipitation Rate to point to version 7 at the bottom of the layer description, as well as some other minor text updates

GPM_3IMERGHHE [doi:10.5067/GPM/IMERG/3B-HH-E/07](https://doi.org/10.5067/GPM/IMERG/3B-HH-E/07)

## How To Test


1. Open with [these URL parameters](http://localhost:3000/?l=Reference_Labels_15m(hidden),Reference_Features_15m(hidden),Coastlines_15m,IMERG_Precipitation_Rate,VIIRS_NOAA21_CorrectedReflectance_TrueColor(hidden),VIIRS_NOAA20_CorrectedReflectance_TrueColor(hidden),VIIRS_SNPP_CorrectedReflectance_TrueColor(hidden),MODIS_Aqua_CorrectedReflectance_TrueColor(hidden),MODIS_Terra_CorrectedReflectance_TrueColor&lg=true&t=2024-07-01-T14%3A59%3A41Z)
2. Click on the "i" and read the layer description, ensure that this sentence is updated with this link "GPM_3IMERGHHE [doi:10.5067/GPM/IMERG/3B-HH-E/07](https://doi.org/10.5067/GPM/IMERG/3B-HH-E/07)"
3. Verify expected result


@nasa-gibs/worldview
